org.flatpak.popout3d.yml

```diff
@@ -0,0 +1,196 @@
+app-id: com.github.PopoutApps.popout3d
+
+runtime: org.freedesktop.Platform
+runtime-version: '19.08'
+sdk: org.freedesktop.Sdk
+
+command: popout3d
+rename-desktop-file: popout3d.desktop
+rename-icon: popout3d_logo
+copy-icon: true
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --filesystem=host:rw
+modules:
+
+  - name: python3-pillow
+    buildsystem: simple
+    build-commands:
+      - pip3 install --no-index --find-links="file://${PWD}" --prefix=${FLATPAK_DEST} Pillow Pillow-PIL
+    sources:
+      - type: file
+        url: https://files.pythonhosted.org/packages/3c/7e/443be24431324bd34d22dd9d11cc845d995bcd3b500676bcf23142756975/Pillow-5.4.1.tar.gz
+        sha256: 5233664eadfa342c639b9b9977190d64ad7aca4edc51a966394d7e08e7f38a9f
+      - type: file
+        url: https://files.pythonhosted.org/packages/5d/e1/d298934cd78984a5e8d616945349ed66a74b9df0337fb1941ce76b8a2b7b/Pillow-PIL-0.1dev.tar.gz
+        sha256: d80ee30805f10bf898df9ce7495f46e88990e8f90b5c2b917722d87526252b2c
+
+  # Hugin deps  
+  - name: glu
+    config-opts:
+      - --disable-static
+    sources:  
+      - type: archive
+        url: https://mesa.freedesktop.org/archive/glu/glu-9.0.0.tar.bz2
+        sha256: 1f7ad0d379a722fcbd303aa5650c6d7d5544fde83196b42a73d1193568a4df12
+    cleanup:
+      - /include
+      - /lib/*.a
+      - /lib/*.la
+      - /lib/pkgconfig
+
+  - name: glew
+    no-autogen: true
+    make-args:
+      - GLEW_PREFIX=/app
+      - GLEW_DEST=/app
+      - LIBDIR=/app/lib
+    make-install-args:
+      - GLEW_PREFIX=/app
+      - GLEW_DEST=/app
+      - LIBDIR=/app/lib
+    sources:
+      - type: archive
+        url: https://downloads.sourceforge.net/project/glew/glew/2.1.0/glew-2.1.0.tgz
+        sha256: 04de91e7e6763039bc11940095cd9c7f880baba82196a7765f727ac05a993c95
+    cleanup:
+      - /include
+      - /lib/pkgconfig
+      - /lib/*.a
+
+  - name: wxWidgets
+    rm-configure: true
+    cleanup:
+      - /bin
+      - /share/bakefile
+    config-opts:
+      - --with-opengl
+      - --with-libjpeg
+      - --with-libtiff
+      - --with-libpng
+      - --with-zlib
+      - --disable-sdltest
+      - --enable-unicode
+      - --enable-display
+      - --enable-propgrid
+      - --disable-webkit
+      - --disable-webview
+      - --disable-webviewwebkit
+      - --with-expat=builtin
+      - --with-libiconv=/usr
+    build-options:
+      cxxflags: "-std=c++0x"
+    sources:
+      - type: archive
+        url: https://github.com/wxWidgets/wxWidgets/releases/download/v3.0.4/wxWidgets-3.0.4.tar.bz2
```

Comment:
does this work with the latest upstream 'stable' wxwidgets? (3.0.5.1)?

Reply:
Changed, will try later.